driver/normalizer/normalizer.go

```diff
@@ -316,6 +506,43 @@ var Normalizers = []Mapping{
 			},
 		),
 	)),
+
+	MapSemantic("Parameter", uast.Argument{}, MapObj(
+		Obj{
+			"Identifier": Obj{
+					// Could be identifier or some keywords like __arg, same fields in any case
+					uast.KeyType: Any(),
+					uast.KeyPos:      Var("id_param_pos"),
+					"LeadingTrivia":  Any(),
+					"TrailingTrivia": Any(),
+					"IsMissing": Bool(false),
+					"Text":      Var("name"),
+					"Value":     Var("name"),
+					"ValueText": Var("name"),
+				},
+			"AttributeLists":     Any(),
```

Comment:
Please don't drop random stuff :sweat_smile: 
I understand why it happens for `IsStructuredTrivia` and other too-detailed things, but this one might be important.

Reply:
Same discussion we had on the C++ driver: semantic objects and specially non-node lists like `Parameter` in this case doesn't have space for anything that's not defined, so I don't know how to handle this, could you fix this in a commit so I can see how you would fix it?

Reply:
None of the fixtures have this field to anything except an empty array. Will assert to `Arr()` for now.